services/121-service/test/payment/fsp-integrations/do-payment-fsp-excel/do-payment-fsp-excel-export-instructions.test.ts

```diff
@@ -104,7 +104,7 @@ describe('Do payment with Excel FSP', () => {
       registrationsProgramWithValidation,
       programIdCbe,
       amount,
-      [TransactionStatusEnum.success],
+      [TransactionStatusEnum.success, TransactionStatusEnum.waiting],
```

Comment:
'Waiting' does not exist in the context of CBE, so we shouldn't just put this here, but keep this focused on what we're actually waiting for.